packages/gatsby-source-wordpress/src/utils/fetch-graphql.ts

```diff
@@ -27,6 +28,13 @@ export const moduleHelpers = {
   },
 }
 
+const errorIs500ish = (e: Error): boolean =>
+  e.message.includes(`Request failed with status code 50`) &&
+  (e.message.includes(`502`) ||
```

Comment:
```suggestion
  [`502`, `503`, `504`].includes(e.message)
  (e.message.includes(`502`) ||
```

Reply:
Ah yep, that's right @j0sh77 ! I think let's leave it as is, it works and reads fine :)

Reply:
Good point!